build_deps.sh

```diff
@@ -38,14 +38,14 @@ function BuildLib()
 
   case $OS in
         Darwin)
-            brew install openssl protobuf boost boost-python log4cxx jsoncpp
+            brew install openssl protobuf boost boost-python boost-python3 log4cxx jsoncpp
```

Comment:
I don't think `boost-python3` is necessary all the time but it shouldn't cause any issues